src/modules/List.js

```diff
@@ -1,53 +1,63 @@
-import ListProperties from './ListProperties.js';
-import verticalDotsIcon from '../img/vertical dots.svg';
-import refresh from '../img/refresh.svg';
+import NewTask from './NewTask.js';
+import ListItem from './List item.js';
 
 export default class List {
-    ListObjects = [
-      new ListProperties(
-        'complete the project',
-        true,
-        0,
-      ),
+  constructor() {
+    this.ListObjects = localStorage.getItem('list') === null ? [] : JSON.parse(localStorage.getItem('list'));
+  }
 
-      new ListProperties(
-        'read the book',
-        true,
-        1,
-      ),
+  addTask() {
+    const addTaskInput = document.querySelector('.add-task');
+    const listform = document.querySelector('.add__task');
+    if (addTaskInput.value.trim().length === 0) return;
+    this.ListObjects.push(new NewTask(addTaskInput.value, false));
+    listform.reset();
+    localStorage.setItem('list', JSON.stringify(this.ListObjects));
+    this.render();
+  }
```

Comment:
- [ ] Adding a new task it adds it with no index in localStorage, it only updates when I add one after it. 
Can you please fix that? 
![image](https://user-images.githubusercontent.com/87689547/161311392-b479f079-a8b2-4472-bfd0-4e7e489078da.png)